### DISCUSSION TITLE
Ideas for Backend Development Roadmap

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [commclassroom/roadmaps#16] [Ideas]
Please add all your suggestions below

**Comment by harshmittal-cyber** [comment]
Please add microservice resources in Backend roadmap. Currently all APIs are built on monolithic architecture in the roadmap

**Reply by kaiwalyakoparkar** [reply]
Thank you for the suggestion. The team is currently working on this. You might see the updated roadmap in 2-3 days. I will notify you here too :)

**Comment by deep1910** [comment]
@kaiwalyakoparkar  i would  like to you involve this nodejs playlist by  THE NET NINJA youtube channel 


https://www.youtube.com/playlist?list=PL4cUxeGkcC9jsz4LDYc6kv3ymONOKxwBU


I found this very much helpful and comprehensive 

also would like if you can involve in Community classroom roadmap for backend web development

this will benefit a lot to the community in learning briefly and 
Node.js docs approach

**Reply by kaiwalyakoparkar** [reply]
Thank you so much. This will be added shortly

**Reply by deep1910** [reply]
Thanks for accepting ,  would like to help the community in more ways possible , in what other terms can i help the community and will help in my growth

**Reply by kaiwalyakoparkar** [reply]
This has been added as Resource 7

**Comment by deep1910** [comment]
Looking out for more contributions
